_includes/php-dev/component-root.md

```diff
@@ -10,7 +10,7 @@ A component's root directory is the top-level directory for that component under
   * For Admin themes, use `app/design/adminhtml`.
   * For language packages, use `app/i18n`.
 
-* `<Magento install directory>/vendor`: This location is found in the alternative setups where the [`composer create-project`]({{ page.baseurl }}/install-gde/prereq/integrator_install.html) command was used to get a Magento 2 metapackage (which downloads the CE or EE code), or a [compressed Magento 2 archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html) was extracted in order to install Magento. 
+* `<Magento install directory>/vendor`: This location is found in the alternative setups where the [`composer create-project`]({{ page.baseurl }}/install-gde/composer.html) command was used to get a Magento 2 metapackage (which downloads the CE or EE code), or a [compressed Magento 2 archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html) was extracted in order to install Magento. 
```

Comment:
Verified this include is used by 2.0.  If used will 404 when clicked in 2.0 pages.

Reply:
Yes, a search for references to these includes in the 2.0 files did return results. I'll make changes.

Reply:
Awesome! Wasn't sure which of these was loading. Thanks, Jeff!